common/types/nipost.go

```diff
@@ -1,27 +1,7 @@
 package types
 
-//go:generate scalegen -types NIPostBuilderState,PoetRequest,PoetServiceID
+//go:generate scalegen
 
 type PoetServiceID struct {
 	ServiceID []byte `scale:"max=32"` // public key of the PoET service
 }
```

Comment:
At the moment I still need the getter for the migration from state-files to DB. (see also my comment there)

Reply:
Right, but you moved the other ones into the migration files. Why keep this one here?

Reply:
I thought that it would cause cyclic dependencies which doesn't seem to be the case so I moved it.

With `v1.4.0` I would like to get rid of all the old code handling state files (`nipost_state*.go` in the `localsql` package) and also remove the data migration from disk to DB. New nodes don't need it and old nodes can upgrade `v1.2.x` -> `v1.3.x` -> `v1.4.x` to keep their state. We can then also remove the `PoetServiceID` method on `HTTPPoetClient`.

Wdyt?